app/services/v2/people/include_params_validator.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module V2
+  module People
+    class IncludeParamsValidator
+      include ActiveModel::Validations
+
+      attr_reader :relationships
+
+      validates_each :relationships, allow_blank: true do |record, _attr, values|
+        unsupported_values = values - ::V2::PersonSerializer::SUPPORTED_RELATIONSHIPS
```

Comment:
Potentially for later, but the only thing that makes this non-generic is the dependency on the `PersonSerializer`